tables/__init__.py

```diff
@@ -0,0 +1,11 @@
+from . import store
```

Comment:
So the top-level package is currently called `tables`... I imagine we'll want to change this to something less namespace-collisiony :-)  Will it be `pdtable`?